include/cppcoro/coroutine.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef CPPCORO_COROUTINE_HPP_INCLUDED
+#define CPPCORO_COROUTINE_HPP_INCLUDED
+
+#if __has_include(<coroutine>)
+
+#include <coroutine>
+
+namespace cppcoro {
+  using std::coroutine_handle;
+  using std::suspend_always;
+  using std::noop_coroutine;
+  using std::suspend_never;
+}
+
+#elif __has_include(<experimental/coroutine>)
+
+#include <experimental/coroutine>
+
+namespace cppcoro {
+  using std::experimental::coroutine_handle;
+  using std::experimental::suspend_always;
+  using std::experimental::suspend_never;
+  using std::experimental::noop_coroutine;
```

Comment:
This seems to break MSVC 2017 (and I think also some MSVC 2019 versions), which doesn't have `noop_coroutine()`.

Maybe put this inside `#if CPPCORO_COMPILER_SUPPORTS_SYMMETRIC_TRANSFER` and add an include of `<cppcoro/config.hpp>`?

Reply:
fixed now thx to @Garcia6l20